src/Models/PaginatedResource.php

```diff
@@ -81,10 +81,17 @@ public function getFirst() {
     }
 
     /**
+     * @return boolean Whether there is a next page
+     */
+    public function hasFirst() {
```

Comment:
Ah sorry didn't see your comment :). I think it's the comment which is wrong since we have a `next` function below.